database/migrations/2022_07_20_225822_add_location_birthday_and_gender_columns_to_users_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table("users", function (Blueprint $table): void {
+            $table->string("location")->nullable();
+            $table->date("birthday")->nullable();
+            $table->string("gender")->nullable();
+        });
+    }
+
```

Comment:
Maybe profile table will be better for storing informations like that. What do u think?